<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGA Shape Grammar</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenCGA"/>
    <addaction name="actionSaveImage"/>
    <addaction name="actionSaveOBJ"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewShadow"/>
    <addaction name="separator"/>
    <addaction name="actionViewBasicRendering"/>
    <addaction name="actionViewSSAO"/>
    <addaction name="actionViewContourRendering"/>
    <addaction name="actionViewLineRendering"/>
    <addaction name="actionViewHatching"/>
    <addaction name="actionViewSketchyRendering"/>
    <addaction name="separator"/>
    <addaction name="actionViewRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionRotationStart"/>
    <addaction name="actionRotationEnd"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCamera"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenCGA">
   <property name="text">
    <string>Open CGA</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionViewShadow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shadow</string>
   </property>
  </action>
  <action name="actionViewBasicRendering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Basic Rendering</string>
   </property>
  </action>
  <action name="actionViewSSAO">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SSAO</string>
   </property>
  </action>
  <action name="actionViewLineRendering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Line Rendering</string>
   </property>
  </action>
  <action name="actionViewSketchyRendering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sketchy Rendering</string>
   </property>
  </action>
  <action name="actionViewRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionGenerateWindowImages">
   <property name="text">
    <string>Generate Window Images (256x256 color)</string>
   </property>
  </action>
  <action name="actionGenerateBuildingImages">
   <property name="text">
    <string>Generate Building Images (256x256 color)</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionGenerateSimpleShapeImages">
   <property name="text">
    <string>Generate Simple Shape Images</string>
   </property>
  </action>
  <action name="actionGenerateLedgeImages">
   <property name="text">
    <string>Generate Ledge Images (256x256 color)</string>
   </property>
  </action>
  <action name="actionGenerateRoofImages">
   <property name="text">
    <string>Generate Roof Images (256x256 color)</string>
   </property>
  </action>
  <action name="actionGenerateRoofImages2">
   <property name="text">
    <string>Generate Roof Images (128x128 grayscale)</string>
   </property>
  </action>
  <action name="actionGenerateWindowImages2">
   <property name="text">
    <string>Generate Window Images (128x128 grayscale)</string>
   </property>
  </action>
  <action name="actionGenerateLedgeImages2">
   <property name="text">
    <string>Generate Ledge Images (128x128 grayscale)</string>
   </property>
  </action>
  <action name="actionGenerateBuildingImages2">
   <property name="text">
    <string>Generate Building Images (128x128 grayscale)</string>
   </property>
  </action>
  <action name="actionViewHatching">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hatching</string>
   </property>
  </action>
  <action name="actionRotationStart">
   <property name="text">
    <string>Rotation Start</string>
   </property>
  </action>
  <action name="actionRotationEnd">
   <property name="text">
    <string>Rotation End</string>
   </property>
  </action>
  <action name="actionSaveGeometry">
   <property name="text">
    <string>Save Geometry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionViewContourRendering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Contour Rendering</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionSaveOBJ">
   <property name="text">
    <string>Save OBJ</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
